pkg/skaffold/errors/err_map.go

```
@@ -87,3 +88,26 @@ var knownBuildProblems = []problem{
 		},
 	},
 }
+
+// Deploy errors in deployment phase
+var knownDeployProblems = []problem{
+	{
+		regexp:  re("(?i).*unable to connect.*: Get (.*)"),
+		errCode: proto.StatusCode_DEPLOY_CLUSTER_CONNECTION_ERR,
+		description: func(err error) string {
+			matchExp := re("(?i).*unable to connect.*Get (.*)")
+			kubeconfig, _ := kubectx.CurrentConfig()
```

don't ignore the error. Please log it at debug level using `logrus.Debugf`